src/java/voldemort/server/PosixHostMatcher.java

```diff
@@ -0,0 +1,19 @@
+package voldemort.server;
+
+import java.util.List;
+
+import voldemort.cluster.Node;
+
+
+public class PosixHostMatcher extends HostMatcher {
```

Comment:
FQDNs are always case insensitive.  This is a property of [the DNS architecture](https://tools.ietf.org/html/rfc1035) and doesn't depend on the OS of any host.

> Note that while upper and lower case letters are allowed in domain
> names, no significance is attached to the case.  That is, two names with
> the same spelling but different case are to be treated as if identical.